pkg/client/oci/oci.go

```diff
@@ -138,6 +205,10 @@ func (r *Repo) getChartDigest(name, version string) (string, error) {
 
 // ListChartVersions lists all versions of a chart
 func (r *Repo) ListChartVersions(name string) ([]string, error) {
+	// If entries is populated use it to list the chart versions
```

Comment:
I'd extend this comment with some of the info you put in the readme, what you explained that if you don't use the index listing versions require checking the tags and so on...